src/solver/optimisation/adequacy_patch.h

```diff
@@ -1,6 +1,6 @@
 /*
-** Copyright 2007-2018 RTE
-** Authors: Antares_Simulator Team
+** Copyright 2007-2022 RTE
+** Authors: RTE-international / Redstork / Antares_Simulator Team
```

Comment:
We should split **adequacy_patch.h/.cpp**  into : 
- **adq_patch_local_matching.h/.cpp**
- **adq_patch_curtailment_sharing.h/.cpp**